site/en/blog/floc/index.md

```diff
@@ -66,21 +72,51 @@ The cohort data made available looks like this:
 ``` json
     {
         "id": "14159",
-        "version": "chrome.1.0"
+        "version": "chrome.2.1"
     }
 ```
 
-The FLoC API is available in Chrome 89 and above, but if you're not taking part in the origin trial, you will need to set flags and run Chrome from the command line. [Run Chromium with flags](http://www.chromium.org/developers/how-tos/run-chromium-with-flags) explains how to do this for different operating systems.  
+The FLoC API is available in Chrome 89 and above, but if your browser is not included in the origin 
+trial, you will need to run Chrome with flags in order to try out the API. [Run Chromium with flags](http://www.chromium.org/developers/how-tos/run-chromium-with-flags) explains how to do this for different operating systems.  
 
-1.  Start Chrome with the following flags:  
+1.  Start Chrome with the following flags. Make sure to copy all the text!<br> <br>  
 
     ``` text
     --enable-blink-features=InterestCohortAPI 
     --enable-features="FederatedLearningOfCohorts:update_interval/10s/minimum_history_domain_size_required/1,FlocIdSortingLshBasedComputation,InterestCohortFeaturePolicy"
     ```
+    <br>
+
+1.  Check that third-party cookies are not blocked and that no ad blocker is running.
+1.  View the demo at [floc.glitch.me](https://floc.glitch.me/) or run the following code from the 
+DevTools console:<br><br>
+
+    ``` js
+    await document.interestCohort()
+    ```
+
+{% Aside %}
+
+The flags used above: 
```

Comment:
Could you embed the relevant sections of the flags in here, e.g.

- `InterestCohortAPI` enables FLoC
- `update_interval/10s` sets the cohort recalculation to be done every 10 seconds. This is only for testing during the 
Origin Trial; the cohort recalculation interval currently defaults otherwise to every seven days.
- `minimum_history_domain_size_required/1` Set the minimum number of domains to be available in order for the cohort to be calculated. This would be higher for normal users.
- `FlocIdSortingLshBasedComputation` sets the clustering algorithm used by FLoC
- `InterestCohortFeaturePolicy` ?

Reply:
I've updated this now, and added an aside about the flags.